movie.js

```diff
@@ -186,6 +189,43 @@ function sortMovies(movieList, sortBy) {
   displayMovies(sortedMovies);
 }
 
+function startCountDown(startingMinutes) {
+  let time = startingMinutes * 60;
+
+  countdown = setInterval(() => {
+    let minutes = Math.floor(time / 60);
+    let seconds = time % 60;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
+    displayCountDown.innerHTML = ` Time left to choose a movie : ${minutes}m : ${seconds}s`;
+    time--;
+    if (time < 0) {
+      clearInterval(countdown);
+      alert("Time's up. Choose a movie FAST !!");
+    }
+  }, 1000);
+}
+
+function calculateTimeSpent() {
+  let hrs = document.getElementById("hours");
+  let mins = document.getElementById("minutes");
+  let secs = document.getElementById("seconds");
+
+  let elapsedTime = 0;
```

Comment:
This works fine, but it could give an unexpected result if the user leaves the tab and the browser pauses the unused tab. To avoid it, you should store the time when the user enters the page and calculate the elapsed time from it. Something like:
```js
const enteringTime = new Date();

setInterval(() => {
  const now = new Date();
  elapsedTime = now - enteringTime;
  // ...
}, 1000);
```

Reply:
Thanks @vdaguenet for the suggestion.
This seems to be the accurate way to find the elapsed time.

Reply:
Exactly!